cookbooks/cdo-ruby/recipes/ruby-build.rb

```diff
@@ -6,14 +6,14 @@
 
 # Arbitrarily use the latest version of ruby build at time this code was
 # written; this probably doesn't matter, but we need to pick something.
-RUBY_BUILD_VERSION = '20221225'.freeze
+RUBY_BUILD_VERSION = '20240727'.freeze
```

Comment:
Each ruby-build version knows how to build certain versions of Ruby. Thus, when you upgrade Ruby you gotta upgrade Ruby Build.

Reply:
I debated myself here whether to split this out into the "pre-upgrade" PR. I think its actually safer to consider the `ruby-build` version upgrade to be part of the Ruby upgrade, and not something to do in advance. My reasoning is that in theory ruby-build being upgraded might change the Ruby 3.0.5 that was being built, and in a sense trigger exactly the scenario we were trying to avoid (having a ruby version upgrade at the same time as the mass pre-upgrade).